### PR TITLE
measurement patch

### PR DESCRIPTION
**Pull request by SansPapyrus683**
_If any of the below don't apply to this Pull Request, mark the checkbox as done._

- [x] I have tested my code.
- [x] I have added my solution according to the steps [here](https://usaco.guide/general/adding-solution#steps).
- [x] I have followed the code conventions mentioned [here](https://usaco.guide/general/adding-solution/#code-conventions), which include the following: 
  - I understand that if it is clear that I have not attempted to follow these guidelines (ex. if I have not used tabs to indent), my PR will be closed.
  - If changes are requested, I will re-request the review after making the changes.

### CODE REVIEW

File: solutions/usaco-763.mdx [hunk 1]
```diff
@@ -2,147 +2,167 @@
...
-			d = in.nextInt(); id = in.nextInt(); c =in.nextInt();
-			L[i] = new Log(d, id, c);
-			ids.put(id, 0);
-		}
-
-		Arrays.sort(L, Comparator.comparingInt(o -> o.d));
-		values.put(0, N+1);
+using std::cout;
+using std::endl;
+using std::vector;
```
Comment (Aadit-Ambadkar) [inline solutions/usaco-763.mdx:24]:
```suggestion
using namespace std;
```

File: solutions/usaco-763.mdx [hunk 2]
```diff
@@ -2,147 +2,167 @@
...
-			int prevCnt = values.get(change);
-			values.put(change, values.get(change)-1);
-			if (values.get(change) == 0) values.remove(change);
+struct Log { int day, cow, change; };
 
-			change += L[i].c;
-			ids.put(L[i].id, change);
-			values.put(change, values.getOrDefault(change,0)+1);
+int main() {
+	std::ifstream read("measurement.in");
```
Comment (Aadit-Ambadkar) [inline solutions/usaco-763.mdx:29]:
```suggestion
	ifstream read("measurement.in");
```

File: solutions/usaco-763.mdx [hunk 3]
```diff
@@ -2,147 +2,167 @@
...
-			else if(isTop){
-				//If it wasn't at the highest but now it is, then we need to increment answer
-				A++;
-			}
-		}
-		PrintWriter out = new PrintWriter(new BufferedWriter(new FileWriter("measurement.out")));
-		out.println(A);
-		out.close();
+	vector<Log> log(n);
+	std::map<int, int> cows;
```
Comment (Aadit-Ambadkar) [inline solutions/usaco-763.mdx:36]:
```suggestion
	map<int, int> cows;
```

File: solutions/usaco-763.mdx [hunk 4]
```diff
@@ -2,147 +2,167 @@
...
-		public BufferedReader reader;
-		public StringTokenizer tokenizer;
-
-		public InputReader() {
-			try {
-				reader = new BufferedReader(new InputStreamReader(System.in), 32768);
-			} catch (Exception e) {
-				throw new NullPointerException("Could not create input stream");
-			}
+	std::sort(
```
Comment (Aadit-Ambadkar) [inline solutions/usaco-763.mdx:41]:
```suggestion
	sort(
```

File: solutions/usaco-763.mdx [hunk 5]
```diff
@@ -2,147 +2,167 @@
...
+		log.begin(), log.end(),
+		[] (const Log& l1, const Log& l2) { return l1.day < l2.day; }
+	);
+
+	std::map<int, int> milk_prod{{g, n}};
```
Comment (Aadit-Ambadkar) [inline solutions/usaco-763.mdx:46]:
```suggestion
	map<int, int> milk_prod{{g, n}};
```

File: solutions/usaco-763.mdx [hunk 6]
```diff
@@ -2,147 +2,167 @@
...
+			/*
+			 * If it wasn't at the highest but now it is,
+			 * then FJ needs to change the portrait also
+			 */
+			changeAmt++;
 		}
+	}
 
-		public boolean hasNextInt() throws IOException { return reader.ready(); }
+	std::ofstream("measurement.out") << changeAmt << endl;
```
Comment (Aadit-Ambadkar) [inline solutions/usaco-763.mdx:84]:
```suggestion
	ofstream("measurement.out") << changeAmt << endl;
```

**Review by Aadit-Ambadkar — APPROVED** [review]
ur going to reject these anyways so might as well approve

**Comment by SansPapyrus683** [comment]
oh wait thought ryan approved already
@ryanchou-dev can you retroactively approve lmao

File: solutions/usaco-763.mdx [hunk 7]
```diff
@@ -2,147 +2,167 @@
...
 			}
-			return tokenizer.nextToken();
+			/*
+			 * if it was the highest and now it's not or there are now
+			 * multiple highest cows, then FJ needs to change the portrait
+			 */
+			changeAmt++;
+		} else if (is_top) {
+			/*
+			 * If it wasn't at the highest but now it is,
```
Comment (ryanchou-dev) [inline solutions/usaco-763.mdx:77]: would prefer casing to be consistent but it's eh

**Review by ryanchou-dev — COMMENTED** [review]
approved**

File: solutions/usaco-763.mdx (hunk 7 above)
Comment (SansPapyrus683) [reply solutions/usaco-763.mdx:77]: will make a pr real quick

**Pull request merged**